templates/resources/common/central-conf/boot.conf.template.j2

```diff
@@ -9,3 +9,4 @@ ssl.mutual=false
 ssl=false
 threads.max=150
 threads.min=30
+xl.spring.cloud.encrypt.key=aaL`;*~`<l-Xv.6
```

Comment:
there is another boot.conf.template.j2 specific for central-configuration. Do we need this change here?

Reply:
This change is not required, forgot to cleanup code, Thanks